### PR TITLE
Enable sandboxing for Android builds to prevent asset double-inclusion

### PR DESCRIPTION
**Pull request by andrewharp**
See https://github.com/bazelbuild/bazel/issues/2334 for context.

### CODE REVIEW

File: tensorflow/tools/ci_build/builds/android.sh [hunk 1]
```diff
@@ -21,4 +21,9 @@ source "${SCRIPT_DIR}/builds_common.sh"
 configure_android_workspace
 
 CPUS=armeabi-v7a,x86_64
-bazel build -c opt --fat_apk_cpu=${CPUS} //tensorflow/examples/android:tensorflow_demo
+
+# Enable sandboxing so that zip archives don't get incorrectly packaged
+# in assets/ dir (see https://github.com/bazelbuild/bazel/issues/2334)
+bazel --bazelrc=/dev/null build -c opt --fat_apk_cpu=${CPUS} \
```
Comment (gunan) [inline tensorflow/tools/ci_build/builds/android.sh]: Could you add a TODO for me to revert these changes once sandboxing is enabled for all builds?

File: tensorflow/tools/ci_build/builds/android_full.sh [hunk 2]
```diff
@@ -51,8 +51,11 @@ do
 done
 
 # Build Jar and also demo containing native libs for all architectures.
+# Enable sandboxing so that zip archives don't get incorrectly packaged
+# in assets/ dir (see https://github.com/bazelbuild/bazel/issues/2334)
 echo "========== Building TensorFlow Android Jar and Demo =========="
-bazel build -c opt --fat_apk_cpu=${CPUS} \
+bazel --bazelrc=/dev/null build -c opt --fat_apk_cpu=${CPUS} \
```
Comment (gunan) [inline tensorflow/tools/ci_build/builds/android_full.sh]: Could you add a TODO for me to revert these changes once sandboxing is enabled for all builds?

**Review by gunan — APPROVED** [review]

**Comment by gunan** [comment]
Jenkins, test this please.

File: tensorflow/tools/ci_build/builds/android_full.sh (hunk 2 above)
Comment (andrewharp) [inline tensorflow/tools/ci_build/builds/android_full.sh]: done

File: tensorflow/tools/ci_build/builds/android.sh (hunk 1 above)
Comment (andrewharp) [inline tensorflow/tools/ci_build/builds/android.sh]: done
